### PR TITLE
drop product picker

### PR DESCRIPTION
**Pull request by adnasa**
#### Summary

- Drop `product-picker-input`

### CODE REVIEW

File: packages/i18n/data/core.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
   "FailedAuthentication.title": "We are unable to authorize you to use the Merchant Center.",
   "NavBar.MCSupport.title": "Support",
   "Notification.hideNotification": "Hide notification",
-  "PageNotFound.paragraph1": "The item you are looking for may have been deleted, does not exist, or the URL was entered incorrectly. Check the URL and try again.\n\nPlease contact your system administrator or the commercetools <a>Help Desk</a> if you have any further questions.",
+  "PageNotFound.paragraph1": "The item you are looking for may have been deleted, does not exist, or the URL was entered incorrectly. Check the URL and try again. Please contact your system administrator or the commercetools <a>Help Desk</a> if you have any further questions.",
```
Comment (emmenko) [inline packages/i18n/data/core.json:56]: Hmm unrelated to this PR, but we should fix this. Let's revert this change please.

Comment (adnasa) [reply packages/i18n/data/core.json]: done

**Review by emmenko — APPROVED** [review]

**Comment by emmenko** [comment]
@adnasa you have to solve the conflicts

**Comment by adnasa** [comment]
haha yeah I'm already on it.

**Pull request merged**